lti_consumer/tests/unit/test_lti_consumer.py

```diff
@@ -22,7 +22,7 @@
 HTML_IFRAME = '<iframe'
 
 
-class TestLtiConsumerXBlock(unittest.TestCase):
+class TestLtiConsumerXBlock(TestCase):
```

Comment:
Was this made necessary because we're checking Django settings now?

Reply:
Yes @ormsbee, that's why.